data/remnant/remnant 2 side missions.txt

```diff
@@ -0,0 +1,146 @@
+			and
+				random < 30
+				has "Remnant: Visit to Paradise 1: defer"
+	on offer
+		conversation
+			`	You are just settling down at a table in the cafeteria when Taely slides onto the bench opposite you. "Greetings, <first>. How is life as an adventurer?"`
+			choice
+				`	"Exciting as always."`
+				`	"I am taking a break right now."`
+			`	She nods sympathetically. "Things have been a bit hectic lately, what with dimensional pockets and mysterious new aliens. I can understand wanting to slow down a bit. With the launch of two new ships in such a short time, I know I feel like I need to just relax a bit." She sighs. "It makes me feel guilty, honestly. I have accomplished so much, and there are so many things to be grateful for, but I even have more projects in development to focus on." She looks lost in thought for a moment.`
+			`	Taely looks up at you. "Could I ask you something about life back in ancestral space?"`
+			choice
+				`	"Sure."`
+					goto askquestion
+				`	"Not right now."`
+			label vaillater
+			`	"Oh, maybe some other time, then." She looks disappointed, and settles in to eat.`
+				defer
+			label askquestion
+			`	"I have been reading through the history archive you brought back, and I came across a mention of Vail." She shakes her head in amusement. "The idea of an entire planet devoted to hoverboarding and leisure activities seems so extravagant... Well, I am curious to visit it. Now, this is not a mission we are asking you to do, so I cannot offer any logistics compensation. I could, however, show you some of our history in exchange. Are you interested?"`
+			choice
+				`	"Yes, that sounds fun."`
+					goto vailfun
+				`	"Sure, I have done a bit of hoverboarding myself."`
+				`	"Maybe later."`
+					goto vaillater
+			apply
+				set "remnant: hoverboarding"
+			`	"Really? The description sounded useful for exploration, and it looked challenging." Her commlink shifts to display a hologram of a hoverboarder executing a jump with an artistic poof of snow around them. "Anyway, I think this trip should be a good diversion for us both. I will meet you onboard the <ship> shortly."`
+				accept
+			label vailfun
+			`	"Excellent. I will meet you onboard the <ship> shortly."`
+				accept
+	on defer
+		set "Remnant: Visit to Paradise 1: defer"
+	on stopover
+		conversation
+			branch science
+				has "Research Laboratory"
+			`As you approach, your navigation system picks up the beacons designating the approach path, and before long you are automatically guided to a landing pad on the outskirts of the main spaceport.`
+				goto vaillanding
+			label science
+			`As you approach Vail, your sensors detect parts of the weather control network tasked with increasing the amount of snowfall on the planet. Your navigation system picks up the beacons designating the approach path, and before too long you are automatically guided down to a landing pad on the outskirts of the main spaceport.`
```

Comment:
```suggestion
			`As you approach Vail, the specialized sensors from your Remnant research laboratory detect parts of the weather control network tasked with increasing the amount of snowfall on the planet. Your navigation system picks up the beacons designating the approach path, and before too long you are automatically guided down to a landing pad on the outskirts of the main spaceport.`
```
Or something like this? Could be nice to note that the reason we're getting this info is the lab.